admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```
@@ -2763,6 +2789,53 @@ $mobile-menu-height: 2.6rem !default;
         }
       }
     }
+
+    .form-group.inline-switch-widget > .input-group {
+      margin-top: 0;
+    }
+
+    #product_details {
+      .form-group.inline-switch-widget > .input-group {
+        margin-top: 0;
+      }
+
+      .switch-widget {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        @include media-breakpoint-up("md") {
+          justify-content: flex-start;
+        }
+
+        h3{
+          margin-bottom: 0;
+        }
+      }
+
+      #product_details_show_condition{
+        margin-top: 0;
+      }
+
+      .customization-field-switch{
```

```suggestion
      .customization-field-switch {
```
Not blocking of course, but I'm surprized the linter doesn't detect this